### PR TITLE
Load Reporting back to using metadata

### PR DESCRIPTION
**Pull request by dgquintas**

### CODE REVIEW

**Comment by ctiller** [comment]
This needs a mechanism to include the load reporting filter or not.

**Comment by dgquintas** [comment]
@ctiller at compile time? There's https://github.com/grpc/grpc/blob/master/src/core/ext/filters/load_reporting/load_reporting.c#L77 for runtime.

File: tools/codegen/core/gen_static_metadata.py [hunk 1]
```diff
@@ -159,6 +160,7 @@
     'user-agent',
     'host',
     'lb-token',
+    'lb-cost-bin',
```
Comment (ctiller) [inline tools/codegen/core/gen_static_metadata.py]: Can this be multivalued? If yes, it doesn't belong here.

Comment (dgquintas) [inline tools/codegen/core/gen_static_metadata.py]: Removed. PTAL.

**Comment by dgquintas** [comment]
Issues: #10576 #9542 

@ctiller can you PTAL?

**Comment by dgquintas** [comment]
Issues: #9542 #10576

**Pull request merged**